### PR TITLE
More 1.19 worldgen mappings

### PR DESCRIPTION
**Pull request by mschae23**
More world generation mappings (mostly for terrain parameters and density functions)

### CODE REVIEW

File: mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping [hunk 1]
```diff
@@ -31,22 +31,38 @@ CLASS net/minecraft/class_6554 net/minecraft/world/biome/source/util/VanillaBiom
...
 		ARG 2 noiseRanges
 	METHOD method_38181 getShoreBiome (II)Lnet/minecraft/class_5321;
+		COMMENT {@return an appropriate shore biome for the given temperature and humidity}.
 		ARG 1 temperature
 		ARG 2 humidity
 	METHOD method_38182 getRegularBiome (IILnet/minecraft/class_6544$class_6546;)Lnet/minecraft/class_5321;
+		COMMENT {@return a regular biome.} This can be a {@linkplain #commonBiomes common} or {@linkplain #uncommonBiomes uncommon} biome, depending on
+		COMMENT temperature and humidity.
+		COMMENT
+		COMMENT Note that for negative weirdness values, only common biomes can get picked by this
```
Comment (apple502j) [inline mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:41]: Is this a new paragraph?

Comment (mschae23) [reply mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:41]: I guess. New paragraphs need the `<p>` tag, right? Should I add that here?

Comment (apple502j) [reply mappings/net/minecraft/world/biome/source/util/VanillaBiomeParameters.mapping:41]: Yeah.

**Review by modmuss50 — APPROVED** [review]
Not my area of expertise, but all seems reasonable.

**Comment by modmuss50** [comment]
Going to merge this, mostly new names so no major worry to breaking things. Thanks 👍

**Pull request merged**